main/res/values-nl/strings.xml

```diff
@@ -558,6 +558,7 @@
   <string name="init_maptrail">Pad tonen</string>
   <string name="init_summary_maptrail">Laat pad op kaart zien</string>
   <string name="init_share_after_export">Open deel-menu na GPX export</string>
+  <string name="init_include_found_status">\"Gevonden\" status mee exporteren</string>
```

Comment:
Do not include translations in your patch, those are handled through crowdin.net.